benchmark/fs/bench-mkdtempSync.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const common = require('../common');
+const fs = require('fs');
+const tmpdir = require('../../test/common/tmpdir');
+
+const bench = common.createBenchmark(main, {
+  type: ['valid-string', 'valid-buffer', 'invalid'],
+  n: [1e3],
```

Comment:
This number of iterations is not too low? Or this benchmark is slow to run?

Reply:
It took a little bit too much for me to test. I'll update it to `1e4`.